### ISSUE TITLE
[BUG]: v0.14.0 hangs when initializing ModelParams

### ISSUE THREAD

**Issue by SpaceAntelope** [SciSharp/LLamaSharp#884]
### Description

I feel that updating GPU drivers might work for most people, but since it's a serious problem and the solution is not necessarily obvious I decided to post a full bug report for anyone afflicted to find. 

The problem appears to be that the result of calling vulkaninfo in **/LLamaSharp/LLama/Native/Load/SystemInfo.cs** as a process and reading the output stream varies a lot. In my case in one machine it hangs, in another it returns empty and in the last one it works fine. Removing **CreateNoWindow = true**  from ProcessStartInfo seems to fix it.

I fixed the one (Win10 22H2) where it kept hanging by updating the GPU drivers which raised the vulkan instance version from 1.2.198 to 1.3.204. The machine (Win11 23H2) where the output stream comes up empty has v1.3.201, but since LLamaSharp accounts for dodgy vulkaninfo outputs this doesn't seem to cause a problem.

Note that vulkaninfo behaves the same when initialized as a System.Diagnostics.Process in a simple console app (i.e. it hangs).

### Reproduction Steps

With .NET8 SDK installed:

repro.csproj:
```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>net8.0</TargetFramework>
    <RootNamespace>vulkan_test</RootNamespace>
    <ImplicitUsings>enable</ImplicitUsings>
    <Nullable>enable</Nullable>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="llamasharp" Version="0.14.0" />
    <PackageReference Include="llamasharp.backend.cpu" Version="0.14.0" />
  </ItemGroup>

</Project>
```

Program.cs:
```
using System.Text.Json;

var modelParams = new LLama.Common.ModelParams("");

var json = JsonSerializer.Serialize(modelParams, new JsonSerializerOptions { WriteIndented = true });

Console.WriteLine(json);
```

**Expected**: Print the default values for ModelParams in json form.
**Actual**: If you're unlucky it hangs forever.

### Environment & Configuration

- Operating system: **Win10**
- .NET runtime version: **8.0.303**
- LLamaSharp version: **0.14.0**
- CUDA version (if you are using cuda backend): **the bug is hit before it attempts to load the model, it doesn't even complain about missing a backend package**
- CPU & GPU device: **Intel Core i7-10700** & **Intel(R) UHD Graphics 630** 
 GPU Driver version: **30.0.101.1273** 
 Vulkan Instance version:  **1.2.198**

### Known Workarounds

You can do any one of the following:
* Update GPU drivers
* While your app is hung, manually stop a process named Vulkan Loader from Task Manager
* While your app is hung, open powershell and do 
```
Get-Process vulkaninfo* | Stop-Process
```
* If you are running from the repo instead of using nuget packages you can go to **/LLamaSharp/LLama/Native/Load/SystemInfo.cs** and comment out **CreateNoWindow = true**  in method **SystemInfo.GetVulkanSummary**

**Comment by SpaceAntelope** [comment]
I'm having a hard time finding the 30.x.x.x driver to roll back to for the time being, sadly.

**Comment by martindevans** [comment]
There have been two versions released since this was reported and the fix from #885 is in place now, so I'll close this issue but please do re-open it if it's still a problem!
